### ISSUE TITLE
[BUG] listContent not exists

### ISSUE THREAD

**Issue by gonoszcsiga** [bolt/bolt#5628]
Hi, my old Bolt 2 template use the listcontent() twig function and i run this error:

```
Error: Uncaught Error: Call to undefined method Bolt\Twig\Handler\RecordHandler::listContent() in /vendor/bolt/bolt/src/Twig/TwigExtension.php:387
```

The listcontent Twig function has been declared in TwigExtension.php, but calling a non-existing function.

Details
-------

 - Relevant Bolt Version: 3.1
 - Install type: trying to update my Bolt 2 to Bolt 3 :)
 - PHP version: 7.0
 - Used webserver: Nginx 1.10.0

**Comment by SahAssar** [comment]
Seems like it was removed in https://github.com/bolt/bolt/commit/66a21490ea9a38ee5a37e1cf2f9c44bfb99c6045

Can't find the PR though, just that commit... Still, it seems like it was just used for backend functions usually, and you shouldn't have to use it for any common frontend stuff...

Can I ask what you used it for?

**Comment by gonoszcsiga** [comment]
It was used in [Bolt 2] (https://github.com/bolt/bolt/blob/release/2.2/app/view/twig/editcontent/_relationships.twig#L31) and we still have this code, because we turn Bolt into webshop system and we hacked the admin as well.  Now I trying to upgrade Bolt 3 and saw this is not only old code error, but also the listContent() cleanup wasn't perfect.

**Comment by SahAssar** [comment]
Ah, in that case it should have been fixed now in #5634, thanks for bringing it to our attention!
